plugin/pkg/auth/authorizer/node/node_authorizer_test.go

```diff
@@ -639,21 +651,23 @@ func BenchmarkWriteIndexMaintenance(b *testing.B) {
 func BenchmarkAuthorization(b *testing.B) {
 	g := NewGraph()
 
-	opts := sampleDataOpts{
+	opts := &sampleDataOpts{
 		// To simulate high replication in a small number of namespaces:
 		// nodes:       5000,
 		// namespaces:  10,
 		// podsPerNode: 10,
-		nodes:                  500,
-		namespaces:             200,
-		podsPerNode:            200,
-		attachmentsPerNode:     20,
-		sharedConfigMapsPerPod: 0,
-		uniqueConfigMapsPerPod: 1,
-		sharedSecretsPerPod:    1,
-		uniqueSecretsPerPod:    1,
-		sharedPVCsPerPod:       0,
-		uniquePVCsPerPod:       1,
+		nodes:                        500,
+		namespaces:                   200,
+		podsPerNode:                  200,
+		attachmentsPerNode:           20,
+		sharedConfigMapsPerNamespace: 0,
+		sharedConfigMapsPerPod:       0,
+		uniqueConfigMapsPerPod:       1,
+		sharedSecretsPerNamespace:    1,
```

Comment:
I think you can drop this... if `shared*PerNamespace` is < `shared*PerPod`, it uses the per-pod number, right?

Reply:
done